dbt/models/model/model.vw_card_res_input.sql

```diff
@@ -654,6 +654,8 @@ LEFT JOIN (
     SELECT *
     FROM forward_fill
     WHERE NOT ind_pin_is_multicard
+        -- Some res parcels are not unique by pin10
+        AND SUBSTR(meta_pin, 11, 4) = '0000'
```

Comment:
**question (non-blocking):** Isn't there a (slim) chance that this will result in NULL values on the filled characteristic/final view if _both_ nearest neighbor PINs?

Reply:
Yes, there is. The other solutions I considered
1. making the two select statements for the NN joins exclude pin14 and use distinct
2. excluding pins from the `nearest_pin_neighbors` macro that are in our res classes and have 14 digit pins as possible neighbors

Those two solutions seemed pretty intense for the level of risk, but i could look into them further.

Reply:
Hmmm, alright. Let's stick with what you've done for now.